### ISSUE TITLE
If the spectrogram is from synthesized waves

### ISSUE THREAD

**Issue by candlewill** [yweweler/single-speaker-tts#9]
Thanks for this great work. The spectrogram graph in the `README` is very good. Does it come from a synthesized wave?  Could you share the generated wave if it is?

![Header](https://github.com/yweweler/single-speaker-tts/blob/master/readme/images/header.png?raw=true)

**Comment by yweweler** [comment]
No this image was produced from the training data and is **not an output of the model itself**.

However, I have uploaded an image pair from an model were I tried to double all parameter sizes inside the model and tried to train it with a large reduction factor `r` of 8.
As one would expect it requires massive amounts of VRAM to train (V100 32GB) and is extremely slow.
While the spectrograms look promising the overall increase in quality is small.
Dropping Griffin-Lim in favor of Wavenet will probably yield even better results with less computational requirements.

![tacotron-doubled-parameters](https://user-images.githubusercontent.com/10332831/50042717-70772a00-0067-11e9-8b2d-97b9d16d94cc.png)

---

On the other hand I train various models to see how the number of mel banks influences the overall synthesis result.
Take a look at the following measurements for the decoder and the post-processing losses for different amounts of banks used. Except from the mel banks I used `r=5` and tried to keep all other parameters equal to the paper.

![tensorflow-feature-comparasion](https://user-images.githubusercontent.com/10332831/50042772-54c05380-0068-11e9-8811-a873e75450f8.png)

Just increasing the number of banks does not actually give you a better system.
As one can see from the post-processing loss, having a higher mel resolution does not improve the linear spectrogram produced by the post-processing stage.
